### ISSUE TITLE
Clean up admin config

### ISSUE THREAD

**Issue by bitbeckers** [rite-of-moloch/RiteOfMoloch-v1.0#153]
* When a RoM instance is created the admin1 and admin2 values are provided in the initdata. 
* If the daoTreasury has the topHat the contract values for admin1 and admin2 are set. Else, the tree is initialized.
* When initializing the tree, the admin1 and admin2 values are not set
* So if a new hats tree is created, admin1 and admin2 are not set in the contract.
* TODO item in contract:     `rm admin vars and replace w/ Hats Protocol subgraph query`

TLDR: when and where can you read the current admins?
